packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -0,0 +1,512 @@
+import * as React from 'react';
+import { IRenderFunction, getDocument } from 'office-ui-fabric-react/lib/Utilities';
+import {
+  DetailsList,
+  DetailsRow,
+  IDetailsRowProps,
+  IDetailsRowStyles,
+  DetailsListLayoutMode,
+  IColumn,
+  ColumnActionsMode
+} from 'office-ui-fabric-react/lib/DetailsList';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { SelectionMode } from 'office-ui-fabric-react/lib/Selection';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { ILinkToken } from 'office-ui-fabric-react/lib/common/DocPage.types';
+import { IApiInterfaceProperty, IApiEnumProperty, IMethod } from './ApiReferencesTableSet.types';
+
+export interface IApiReferencesTableProps {
+  title?: string;
+  properties: IApiInterfaceProperty[] | IApiEnumProperty[];
+  methods?: IMethod[];
+  renderAsEnum?: boolean;
+  renderAsClass?: boolean;
+  key?: string;
+  name?: string;
+  description?: string;
+  extendsTokens?: ILinkToken[];
+}
+
+export interface IApiReferencesTableState {
```

Comment:
This component shouldn't use state, because you never update the state. Instead, just store the relevant things in private members (and extend React.PureComponent rather than React.Component).